### DISCUSSION TITLE
Autocomplete does not work

### DISCUSSION THREAD

**Discussion by LeFede** [lapce/lapce#1502] [Q&A]
Hello! My autocomplete doesn't work. I might be being just a newbie here. 
Is there something I'm missing to turn it on? (ctrl + space doesn't work either)

![image](https://user-images.githubusercontent.com/35147097/195649569-c11856e6-19b5-4903-843c-d2099a61c044.png)

Running on Crystal Linux (arch based)
![image](https://user-images.githubusercontent.com/35147097/195649935-d68c54c3-630b-4f2c-97dc-8aaded7e8048.png)

**Comment by codespy** [comment]
I have the same problem with python on Windows 11.
Is there possibility  to enable code completion?
Tested on 0.2.1

**Reply by panekj** [reply]
Ask plugin maintainer, it's not in our scope

**Comment by rockiger** [comment]
I am very new to lapce, so take my help with a grain of salt.

Which version of lapce do you use, the tarball, flatpak or snap? Do you the typescript plugin installed and the npm typescript-language-server? 

For me autocompletion didn't work until I used the tarball which had access to my node environment.
